Social competence survey for bonus points

juandpenan opened the pull request:
** Note: Your contribution is expected to meet the conventions and policies described in the [contribution guidelines](https://github.com/RoboCupAtHome/RuleBook/wiki/Guidelines:-Contributing) **

## Description

Closes issue #827 

Changes proposed in this pull request:
- Introduced bonuses for receptionist and restaurant tasks.
- Assigned qualitative assessment scores ranging from 0 to 50 based on a survey for teams.
- Mandated each team to bring a minimum of two public evaluators and inform the referee beforehand.
- Provided a Microsoft Form for surveys and an online spreadsheet for result visibility.

johaq commented on general_rules/PenaltiesBonuses.tex:
```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
 	\item It is the decision of the \iaterm{Technical Committee}{TC} if (and to which degree) the bonus score is granted.
 \end{enumerate}
 
+\subsection{Bonus for perceived social intelligence of the robot}\label{rule:perceived_intelligence}
```
Remove "of the robot"

johaq commented on general_rules/PenaltiesBonuses.tex:
```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
 
+\subsection{Bonus for perceived social intelligence of the robot}\label{rule:perceived_intelligence}
+\begin{enumerate}	
+    \item For the task \iterm{Receptionist} in \iterm{Stage~I} and, \iterm{Restaurant} in \iterm{Stage~II} tests. Teams are allowed to request an assessment of the robot's performance regarding its perceived social intelligence.
```
For the tests Receptionist... ~~tests~~

johaq commented on general_rules/PenaltiesBonuses.tex:
```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
+\begin{enumerate}	
+    \item For the task \iterm{Receptionist} in \iterm{Stage~I} and, \iterm{Restaurant} in \iterm{Stage~II} tests. Teams are allowed to request an assessment of the robot's performance regarding its perceived social intelligence.
+    
+    \item This bonus, ranging from 0 to 50, depends on the robot's social performance. Referees and task volunteers will assess the task's social performance using specially designed scales in a survey.
```
depends on the robot's social performance which will be assessed by Referees...

juandpenan commented:
> Like the idea. Couple of notes:
> 
> * I'm not sure we want the referee to evaluate since they should be focusing on running the test
> * Probably need to keep the group of volunteers consistent for one test block for fairness in evaluation
> 
> Aside from the notes please change task to test which is the usual terminology in the rulebook (I know not 100% consistent everywhere but it should be)
Thank you, @johaq, for the review! I've incorporated the suggested changes and added an additional item based on our discussion in the last meeting.  " The referee has the authority to skip the social assessment test if they believe the robot's performance is not suitable for measurement."

johaq commented on general_rules/PenaltiesBonuses.tex:
```diff
@@ -42,18 +42,18 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
+    \item For the test \iterm{Receptionist} in \iterm{Stage~I} and, \iterm{Restaurant} in \iterm{Stage~II} tests. Teams are allowed to request an assessment of the robot's performance regarding its perceived social intelligence.
     
-    \item This bonus, ranging from 0 to 50, depends on the robot's social performance. Referees and task volunteers will assess the task's social performance using specially designed scales in a survey.
+    \item This bonus, ranging from 0 to 50, depends on the robot's social performance which will be assessed by Referees using a specially designed scales in a survey.
```
scale

johaq commented on general_rules/PenaltiesBonuses.tex:
```diff
@@ -42,18 +42,18 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
-    
-    \item Teams requesting a social assessment are both required and responsible for bringing a minimum of two individuals from the general public to evaluate their performance. Evaluators from the same team re not allowed.
-    
+    \item Every team seeking a social assessment should inform the referee before the task. that they want to be evaluated using perceived social intelligence.    
```
...before the test that...

johaq reviewed (changes requested):
Two small typos.
Otherwise looks good to me.

juandpenan commented:
> Two small typos. Otherwise looks good to me.

Done! Thanks a lot

akinobu1998 commented:
The form and spreadsheet look like owned by a personal account and the spreadsheet is private.
Is this URL and QR code the finalized version? If not, we can distribute the QR code via Telegram or printed QR code.
Sometimes online document disappears by accident, and I also worry about spam. Then we had to change the link.

I think we should also include the question list in the rulebook to track changes and review the questions more easily.

johaq commented:
I'm not familiar with office forms and if you can host them on your own. Ideally the form would be on the athome website or hosted by a University I think.

LeroyR commented on general_rules/PenaltiesBonuses.tex:
```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
 
+\subsection{Bonus for perceived social intelligence}\label{rule:perceived_intelligence}
+\begin{enumerate}	
+    \item For the test \iterm{Receptionist} in \iterm{Stage~I} and, \iterm{Restaurant} in \iterm{Stage~II} tests. Teams are allowed to request an assessment of the robot's performance regarding its perceived social intelligence.
```
Imho should be mandatory and integrated in the scoring of the tasks - to make sure league/referee prepares volunteers.
To be useable data we have to define a minimum number of respondents, 
for testing the procedure (or getting an overview of the league) in local leagues we can always opt to skip parts of a task e.g. this survey.

LeanderVonSeelstrang replied:
I don't think it has to be mandatory, but I agree, that it should be integrated in the scoring of the tasks. Since you can't lose anything anyway, any rational team should always opt for social scoring.

johaq replied:
Yes, I think we can just make it mandatory

LeroyR replied:
~~Teams are allowed~~ Teams are assessed

LeroyR commented on general_rules/PenaltiesBonuses.tex:
```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
+
+    \item Every team seeking a social assessment should inform the referee before the test. that they want to be evaluated using perceived social intelligence.    
+  
+    \item After the test is completed, the evaluators will fill out the form accessible via the QR code in Figure \ref{fig:qr-survey}, or through this \href{https://forms.office.com/Pages/ResponsePage.aspx?id=6sSEXw03nkuDDHVvi_G1H7VNGCdGFtZJs0ryJVVWtCFUQVFSWDlYM0FHRVA2QllIT0tOQjI2QUcxQi4u}{link}.
```
Need to be hosted somewhere under control of the league. Fallback sheet in this github for printing.

johaq replied:
We can figure out hosting later

LeroyR commented on general_rules/PenaltiesBonuses.tex:
```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
+    
+    \item The referee has the authority to skip the social assessment test if they believe the robot's performance is not suitable for measurement.
+    
+    \item The score will be automatically recorded in this \href{https://urjc-my.sharepoint.com/:x:/r/personal/juan_pena_urjc_es/Documents/ROBOCUP%20@HOME%20PSI%20SCALE%20PROPOSAL.xlsx?d=wfdc816bee34742e1a9e5bea95677985d&csf=1&web=1&e=zRwl4u}{spreadsheet}.
```
Need to be hosted somewhere under control of the league. Fallback script maybe? Link or document how to calculate score atleast.

LeanderVonSeelstrang replied:
I also think we need to mention how the score is calculated. Average value from all 15 ratings collected times 10, so that a maximum of 50 points can be achieved.

$$\text{Social Score} = \text{scaling} \times \frac{1}{n} \sum_{i=1}^{n} \text{Rating}_i$$

$$\text{scaling} = \frac{\text{max Social Score}}{\text{max Rating}} = 10$$

$$\text{n} = \| \text{{Rating}} \|$$

LeanderVonSeelstrang commented on general_rules/PenaltiesBonuses.tex:
```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
+\begin{enumerate}	
+    \item For the test \iterm{Receptionist} in \iterm{Stage~I} and, \iterm{Restaurant} in \iterm{Stage~II} tests. Teams are allowed to request an assessment of the robot's performance regarding its perceived social intelligence.
+    
+    \item This bonus, ranging from 0 to 50, depends on the robot's social performance which will be assessed by Referees using a specially designed scale in a survey.
```
I think we should name the categories that are asked (although this makes the rulebook longer). Not only referees, but also others who only fill out this survey for a task should be allowed. Also, I don't think we've designed a specific scale, but maybe I'm just misunderstanding. The survey should also be linked here. 

1.
This bonus, which ranges from 0 to 50, depends on the robot's social performance, which is assessed by a group of evaluators (constant for the task) using a \href{https://forms.office.com/Pages/ResponsePage.aspx?id=6sSEXw03nkuDDHVvi_G1H7VNGCdGFtZJs0ryJVVWtCFUQVFSWDlYM0FHRVA2QllIT0tOQjI2QUcxQi4u}{survey}. The following categories are recorded on a linear scale of 1-5 and all values are averaged: Perception, social interaction, recognition, observation, adaptation, prediction, support, trustworthiness, empathy, harm avoidance, social awareness, arrogance, learning, manners, social competence.

2.
This bonus, which ranges from 0 to 50, depends on the robot's social performance, which is assessed by a group of evaluators (constant for the task) using a \href{https://forms.office.com/Pages/ResponsePage.aspx?id=6sSEXw03nkuDDHVvi_G1H7VNGCdGFtZJs0ryJVVWtCFUQVFSWDlYM0FHRVA2QllIT0tOQjI2QUcxQi4u}{survey}. The rated categories are recorded on a linear scale from 1-5 and all values are then averaged.

johaq replied:
Yes please add a scoring item to the tests' scoresheets.
I think it's fine if the form is linked once...

LeanderVonSeelstrang commented on general_rules/PenaltiesBonuses.tex:
```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
+    
+    \item This bonus, ranging from 0 to 50, depends on the robot's social performance which will be assessed by Referees using a specially designed scale in a survey.
+
+    \item Every team seeking a social assessment should inform the referee before the test. that they want to be evaluated using perceived social intelligence.    
```
Small typo. A point in the middle after test. Also should -> must. Maybe also request how mich before. E.g. an hour.

Any team seeking a social assessment must inform the referee at least one hour before the test that they wish to be evaluated on perceived social intelligence.

johaq replied:
Should actually be the day before the test in the team leader meeting

LeanderVonSeelstrang commented on general_rules/PenaltiesBonuses.tex:
```diff
@@ -42,6 +42,26 @@ \subsection{Bonus for outstanding performance}\label{rule:outstanding_performanc
+  
+    \item After the test is completed, the evaluators will fill out the form accessible via the QR code in Figure \ref{fig:qr-survey}, or through this \href{https://forms.office.com/Pages/ResponsePage.aspx?id=6sSEXw03nkuDDHVvi_G1H7VNGCdGFtZJs0ryJVVWtCFUQVFSWDlYM0FHRVA2QllIT0tOQjI2QUcxQi4u}{link}.
+    
+    \item The referee has the authority to skip the social assessment test if they believe the robot's performance is not suitable for measurement.
```
But is that fair? If the team wants to be evaluated, they should be allowed to get it. The referee can suggest it, but I think the team should be allowed to decide.

LeanderVonSeelstrang reviewed (changes requested):
Generally good, but it should be fully integrated into the task and the method of converting the assessment into a score should be explicitly stated. Also the survey should always be linked. Possibly a non-editable version.

juandpenan commented:
Thanks for the feedback. So I modify the following:

- Added [questions](https://github.com/RoboCupAtHome/RuleBook/blob/8deb3d0ea964947045e9c5ee113ee449f55d21f6/general_rules/PenaltiesBonuses.tex#L53C1-L54C12)
- Added the score calculation [method](https://github.com/RoboCupAtHome/RuleBook/blob/8deb3d0ea964947045e9c5ee113ee449f55d21f6/general_rules/PenaltiesBonuses.tex#L71)
- Made the score "mandatory" for [receptionist](https://github.com/RoboCupAtHome/RuleBook/blob/8deb3d0ea964947045e9c5ee113ee449f55d21f6/scoresheets/Receptionist.tex#L12) and [restaurant](https://github.com/RoboCupAtHome/RuleBook/blob/8deb3d0ea964947045e9c5ee113ee449f55d21f6/scoresheets/Restaurant.tex#L14) tests
- Also in the upcoming days will look for different options to host the survey.

Thanks!

LeroyR reviewed (approved):
lgtm, slight wording change needed as assessment is mandatory
